webapp/src/components/properties/user/user.tsx

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react'
+import Select from 'react-select'
+
+import {IUser, WorkspaceUsersContext} from '../../../user'
+
+import './user.scss'
+import {getSelectBaseStyle} from '../../../theme'
+
+type Props = {
+    value: string,
+    readonly: boolean,
+    onChange: (value: string) => void,
+}
+
+const UserProperty = (props: Props): JSX.Element => {
+    let value: IUser | undefined
+
+    if (props.readonly) {
+        return (<div className='UserProperty octo-propertyvalue'>{value ? value.username : props.value}</div>)
+    }
+
+    return (
+        <WorkspaceUsersContext.Consumer>
```

Comment:
Is there any reason to not use https://reactjs.org/docs/hooks-reference.html#usecontext, I think would be more clear to use just:

```typescript
const workspaceUsers = useContext(WorkspaceUsersContenxt);
```

Reply:
Done. Learning a lot about React hooks!